### PR TITLE
Support ubuntu bionic source builds.

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/3917

### CODE REVIEW

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -305,6 +305,11 @@ case $(uname -s) in
                         echo "Installing solidity dependencies on Ubuntu Artful (17.10)."
                         install_z3="libz3-dev"
                         ;;
+                    bionic)
+                        #artful
```
Comment (axic) [inline scripts/install_deps.sh]: artful?

File: Changelog.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 Features:
  * SMTChecker: Integration with CVC4 SMT solver
+ * Build system: Support ubuntu bionic.
```
Comment (axic) [inline Changelog.md]: Please capitalise the same was in the script.

**Review by axic — CHANGES_REQUESTED** [review]
As above.

**Comment by chriseth** [comment]
Updated.

**Review by axic — APPROVED** [review]

**Pull request merged**